regions/shapes/circle.py

```diff
@@ -39,7 +39,11 @@ def area(self):
 
     def contains(self, pixcoord):
         pixcoord = PixCoord._validate(pixcoord, name='pixcoord')
-        return self.center.separation(pixcoord) < self.radius
+        in_circle = self.center.separation(pixcoord) < self.radius
+        if self.meta.get('inverted', False) is False:
```

Comment:
Generally, we should avoid `is False` checks unless `inverted` could be `None`, which I don't think can happen.  This is probably better as `if not self.meta.get('inverted', False):`

Reply:
You could also simplify this a bit by inverting the if statement (`if self.meta.get('inverted', False): return not...`)

Reply:
I did this to cover the `None` case. But if you think it's unnecessary , i will change it.